.github/ISSUE_TEMPLATE/use-question.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Use question
```

Comment:
Can we call this "Usage Question" instead?

Reply:
yeah I'll change it